src/pybind/rados.py

```diff
@@ -1947,6 +2156,13 @@ def retfunc(self, *args, **kwargs):
         return retval
     return retfunc
 
+class OmapItem(object):
+    def __init__(self, key, value, lens):
+        self.key = key
+        self.value = value
+        self.lens = lens
```

Comment:
could you add an empty line between functions? see https://www.python.org/dev/peps/pep-0008/#blank-lines